### ISSUE TITLE
launch and task.json for razer tutorial

### ISSUE THREAD

**Issue by snlacks** [aspnet/AspNetCore.Docs#12136]
When using dotnet core command line sdk with VS Code, the launch.json needs to be updated with the framework version number.
```
            "program": "${workspaceFolder}/bin/Debug/netcoreapp2.2/RazorPagesMovie.dll",`
```

and tasks.json 

```
{
    // See https://go.microsoft.com/fwlink/?LinkId=733558
    // for the documentation about the tasks.json format
    "version": "2.0.0",
    "tasks": [
        {
            "label": "build",
            "command": "dotnet build",
            "type": "shell",
            "group": "build",
            "presentation": {
                "reveal": "silent"
            },
            "problemMatcher": "$msCompile"
        }
    ]
```
}

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a25bf940-8977-fec4-461b-542a9f550fc8
* Version Independent ID: 5492c641-af15-64b8-beaa-ddf511be73e6
* Content: [Tutorial: Get started with Razor Pages in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/razor-pages-start?view=aspnetcore-2.2&tabs=visual-studio#feedback)
* Content Source: [aspnetcore/tutorials/razor-pages/razor-pages-start.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/tutorials/razor-pages/razor-pages-start.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-tutorials**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by Rick-Anderson** [comment]
Why are you using the Framework and not ASP.NET Core?  What's not working?  Are you saying the following doesn't work?

```
dotnet new webapp -o RazorPagesMovie
code -r RazorPagesMovie
```

**Comment by snlacks** [comment]
I'm not.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by Rick-Anderson** [comment]
@scottaddie  can we get an include or something else with the additional instructions for VSC?

**Comment by scottaddie** [comment]
@snlacks Can you please provide the following pieces of info to aid in troubleshooting?

1. Which version of the C# extension are you using in VS Code?
2. What do you see when you run `dotnet --version` in this project's directory?

**Comment by snlacks** [comment]
I was using netcoreapp2.2, I don't know the exact version that would have been or the C# extension version for VS Code; but it would have been up-to-date/latest for 16 days ago. It would have been on Windows 10 (given the date and time of the status and where I was) 

Re-reading my initial issue, I want to clarify that the above code wasn't being inserted into those files and needed to be added in order to get the rest of the instructions in the tutorial working. If I remember correctly, I had to create tasks.json.

**Comment by scottaddie** [comment]
@snlacks Without the 2 requested pieces of info, it's going to be difficult to troubleshoot. You can find the C# extension version number on the **Extensions** view in VS Code:

![image](https://user-images.githubusercontent.com/10702007/57536093-170ae280-7309-11e9-9b8d-6e80611244c0.png)

You can find the .NET Core SDK version number by running `dotnet --version` in VS Code's **Integrated Terminal**. Make sure you run the command in your project's root directory (where the *.csproj file exists):

![image](https://user-images.githubusercontent.com/10702007/57536263-68b36d00-7309-11e9-8668-69187156e8e7.png)

**Comment by snlacks** [comment]
I confirmed that it is 2.2.203  and C# extension 1.19.1. This time I watched what I did a bit more closely.

1. I ran the commands to create the app and open code in that project.
2. I used start debugging with .NET Core to create the launch.json
3. I added a configuration for .NET Core web app. This added the configuration to launch the app.
4. I change <target-framework>/<project-name.dll> to netcoreapp2.2/RazorPagesMovie.dll
